docs/installation.md

```diff
@@ -188,4 +186,7 @@ The command output contains installation instructions. For example, to install t
 openapi completion >> ~/.bashrc
 ```
 
-The approach is similar for other shells. After the installation, restart your terminal for changes to take effect.
+The approach is similar for other shells. After the installation, restart your terminal for the changes to take effect.
+
+## What to do next
+Go back to the [Quickstart guide](./docs/quickstart.md) and finish it.
```

Comment:
issue: The quickstart guide should not require the user to switch back and forth between pages. In this section, we should recommend going to the configuration file and learning how to adjust it, and/or reading more about the supported commands and using them.